app/controllers/issues/socializations_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Issues
+  class SocializationsController < ApplicationController
+    def follow
+      @issue = Issue.find(params[:issue_id])
+      current_user.follow!(@issue)
```

Comment:
in case it return false? Maybe we should show message that something went wrong?